lib/sidekiq/cloudwatchmetrics.rb

```diff
@@ -258,6 +258,9 @@ def publish
 
       # We can only put 20 metrics at a time
       metrics.each_slice(20) do |some_metrics|
+        if @interval_s < 60
+          metrics.each { |metric| metric.merge!(storage_resolution: 1) }
```

Comment:
[StorageResolution accepts values of 1 or 60](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_MetricDatum.html)